pkg/tests/upgrade_tests/upgrade_test.go

```diff
@@ -723,9 +716,10 @@ func testPrereleaseExtensionUpgrade(t *testing.T) {
 		}
 		t.Logf("successfully tested extension upgrade flow with --upgrade-prereleases-extensions true")
 	}()
+	os.Exit(0)
 }
 
-func testExtMigrationFailure(t *testing.T) {
+func TestExtMigrationFailure(t *testing.T) {
```

Comment:
do we not need a new image here? I think we do so we need to do a buildPromscaleImageFromRepo(t) in here too.

Reply:
We need but Test functions are executed in sequence the previous Test func is already building the image & it's available for this test as well. So we are skipping the build process which saves 2-3 mins in running the tests. 

The tests are succeeding because the previously built image is available for the current Test func as well.